### PR TITLE
Migration to include uuid on system_events table

### PR DESCRIPTION
**Pull request by lavabyrd**

### CODE REVIEW

File: store/postgresql/system_event.go [hunk 1]
```diff
@@ -66,6 +66,18 @@ func (d *Driver) GetSystemEvents(ctx context.Context, params structs.SystemEvent
 		i++
 	}
 
+	if params.Id != "" {
+		whereC = append(whereC, ` id =  $`+strconv.Itoa(i))
+		args = append(args, params.Id)
+		i++
+	}
+
+	if params.Validator != "" {
```
Comment (pacmessica) [inline store/postgresql/system_event.go:75]: so difference between `params.Validator` and `params.Address` is that validators don't need to be converted to an address (ie ` common.HexToAddress(params.Address).Hash().Big().String()` on line 65)?

You tested this out?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,64 +1,95 @@
 # Change Log
 
+## [0.0.10] - 2021-07-05
+
+### Added
+
+- Adds `validator` lookup for sender and recipient ID to return validator specific events
```
Comment (pacmessica) [inline CHANGELOG.md:7]:
```suggestion
- Adds `validator`param lookup to `/system_events` for sender and recipient ID to return validator specific events
```

**Review by pacmessica — COMMENTED** [review]
I don't think we support queries by id for system events in other networks. Is this something particular to skale? or should we be adding this as a default?

File: store/postgresql/system_event.go (hunk 1 above)
Comment (lavabyrd) [reply store/postgresql/system_event.go:75]: > I don't think we support queries by id for system events in other networks. Is this something particular to skale? or should we be adding this as a default?

yep, with Near we have it available to show a page for individual events using `/events/:id` . It allows us to show pages like this: https://hubble.figment.io/near/chains/mainnet/events/741 

Its got a flaw in that we don't have a way of ensuring the ID is the same in the case of a table drop/reindexing but Hubble sends out those event urls in tweets/emails etc. Ideally we should have a generated unique hash as the ID that would be constant each time we index that exact event but it'll be something we should be looking at in the future

File: client/transport/webapi/request.go [hunk 3]
```diff
@@ -204,11 +204,13 @@ type ValidatorStatisticsParams struct {
...
-	Address    string `json:"address"`
-	SenderID   uint64 `json:"sender_id"`
-	ReceiverID uint64 `json:"receiver_id"`
-	Limit      uint64 `json:"limit"`
-	Offset     uint64 `json:"offset"`
+	After      uint64  `json:"after"`
+	Kind       string  `json:"kind"`
+	Address    string  `json:"address"`
+	Validator  string `json:"validator"`
+	Id         string  `json:"id"`
```
Comment (lukanus) [inline client/transport/webapi/request.go:211]: As pacmessica mentioned below. ID :)

File: client/transport/webapi/handler.go [hunk 4]
```diff
@@ -810,6 +810,8 @@ func (c *Connector) GetSystemEvents(w http.ResponseWriter, req *http.Request) {
 	params := SystemEventParams{}
 	switch req.Method {
 	case http.MethodGet:
+		params.Id = req.URL.Query().Get("id")
+		params.Validator = req.URL.Query().Get("validator")
```
Comment (lukanus) [inline client/transport/webapi/handler.go:814]: those system events were meant to store various kinds of ID, maybe we can find some general term for that. Also you are querying id so validator_id would be more appropriate. But maybe something like party_id (as refer both to sender and receiver) would do ?  
Kind reminder that you may also like to use types in the query. so it won't break when someone starts adding different kind of events with other type of ids.

Comment (lavabyrd) [reply client/transport/webapi/handler.go:814]: I think `validator_id` makes more sense so there's no confusion if we added node ID etc. I'll look into the types in the query, not familiar with it but I'll research, thanks!

File: store/postgresql/system_event.go (hunk 1 above)
Comment (lavabyrd) [reply store/postgresql/system_event.go:75]: > so difference between `params.Validator` and `params.Address` is that validators don't need to be converted to an address (ie ` common.HexToAddress(params.Address).Hash().Big().String()` on line 65)?
> 
> You tested this out?

yup. tested all of them and work as expected.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,64 +1,95 @@
 # Change Log
 
+## [0.0.10] - 2021-07-05
```
[the remark this answers is not in the document]
Comment (lavabyrd) [reply CHANGELOG.md]: As discussed on Slack, sounds good! Done

**Review by pacmessica — APPROVED** [review]
lgtm :)

**Pull request merged**